### ISSUE TITLE
Cube Store segfaults on M1

### ISSUE THREAD

**Issue by Perlence** [cube-js/cube.js#4474]
The Cube Store executable bundled with the cubejs/cube image doesn't run on an Apple M1 machine. I'd like to run Cube in the development mode as a Docker container.

```sh
$ docker run --rm -it cubejs/cube:v0.29.53 bash
root@c21b9b9af83a:/cube/conf# /cube/node_modules/@cubejs-backend/cubestore/downloaded/latest/bin/cubestored
Segmentation fault
```

However, unpacking the executable fixes the issue:

```sh
root@c21b9b9af83a:/cube/conf# apt update
[…]
root@c21b9b9af83a:/cube/conf# apt install upx
[…]
root@c21b9b9af83a:/cube/conf# upx -d /cube/node_modules/@cubejs-backend/cubestore/downloaded/latest/bin/cubestored
[…]
root@c21b9b9af83a:/cube/conf# /cube/node_modules/@cubejs-backend/cubestore/downloaded/latest/bin/cubestored
2022-05-02 16:58:23,253 INFO [cubestored] <pid:291> Cube Store version 0.29.49
2022-05-02 16:58:23,263 INFO [cubestore::http::status] <pid:291> Serving status probes at 0.0.0.0:3031
2022-05-02 16:58:23,267 INFO [cubestore::metastore] <pid:291> Creating metastore from scratch in /cube/conf/.cubestore/data/metastore
2022-05-02 16:58:23,288 INFO [cubestore::mysql] <pid:291> MySQL port open on 0.0.0.0:3306
2022-05-02 16:58:23,288 INFO [cubestore::http] <pid:291> Http Server is listening on 0.0.0.0:3030
2022-05-02 16:58:23,292 INFO [cubestored] <pid:322 sel1> Cube Store version 0.29.49
2022-05-02 16:58:23,297 INFO [cubestored] <pid:324 sel2> Cube Store version 0.29.49
2022-05-02 16:58:23,302 INFO [cubestored] <pid:337 sel3> Cube Store version 0.29.49
2022-05-02 16:58:23,306 INFO [cubestored] <pid:346 sel4> Cube Store version 0.29.49
^C2022-05-02 16:58:25,675 INFO [cubestored] <pid:291> Received Ctrl+C, shutting down.
```

**Version:**

- macOS v12.3.1
- Docker Desktop v4.6.1 with the new virtualization framework and VirtioFS
- Cube v0.29.53

**Comment by Perlence** [comment]
Many thanks!

**Comment by ovr** [comment]
Thank you @Perlence for bug reporting! You are right, something went wrong with UPX. It started to break the ELF binary format. As a Quick solution, I disabled it for now.

This fix will be in the next upcoming release.

**Comment by ovr** [comment]
Ok, I see. The latest release didn't bump the version of cubestore.

**Comment by ovr** [comment]
Tested & Fixed in `v0.29.56`.
